airflow/executors/kubernetes_executor.py

```diff
@@ -40,12 +42,18 @@
 from airflow.exceptions import AirflowConfigException, AirflowException
 from airflow.utils.log.logging_mixin import LoggingMixin
 
+MAX_POD_ID_LEN = 253
+MAX_LABEL_LEN = 63
+
 
 class KubernetesExecutorConfig:
-    def __init__(self, image=None, image_pull_policy=None, request_memory=None,
+    """Configuration for Kubernetes Executor"""
+    def __init__(self,  # pylint: disable=too-many-arguments
+                 image=None, image_pull_policy=None, request_memory=None,
                  request_cpu=None, limit_memory=None, limit_cpu=None, limit_gpu=None,
                  gcp_service_account_key=None, node_selectors=None, affinity=None,
                  annotations=None, volumes=None, volume_mounts=None, tolerations=None, labels=None):
+        """Initializes the class"""
```

Comment:
```suggestion
```
It is not necessary. We should keep the documentation in the class description

Reply:
Done.